bokeh/layouts.py

```diff
@@ -315,9 +315,16 @@ def gridplot(
             toolbar_options = {}
         if 'toolbar_location' not in toolbar_options:
             toolbar_options['toolbar_location'] = toolbar_location
+
+        # Fixed sizing mode needs scale_width for the toolbar
+        # for layout to work correctly.
+        if sizing_mode == 'fixed':
+            toolbar_sizing_mode = 'scale_width'
```

Comment:
Another thought is that this is a hack. It shouldn't be the final solution. What's there is the "minimum viable hack" which I think is the best way to do it.